web/js/components/measure-tool/measure-tooltip.js

```diff
@@ -101,16 +104,34 @@ export default function MeasureTooltip(props) {
     }
   };
 
-  return (
-    <div className={`tooltip-measure tooltip-custom-black ${activeStaticClass}`}>
-      <span dangerouslySetInnerHTML={{ __html: getMeasurementValue() }} />
-      {!active && (
-        <span className="close-tooltip" onClick={onRemove} onTouchEnd={onRemove}>
-          <FontAwesomeIcon icon="times" fixedWidth />
-        </span>
-      )}
-    </div>
-  );
+  const checkGeographicCoordValidity = (val) => val.indexOf('NaN') < 0;
```

Comment:
Instead of having to add all this code here, couldn't you just modify the `drawChangeListener` in `ol-measure-tooltip` to check the `tooltipCoord` in a similar way to how you are checking the mouse coordinates in the `condition` function that was added to the draw interaction?  For example: 
```
// ol-measure-tool line 200
    drawChangeListener = feature.getGeometry().on('change', (e) => {
      const geom = e.target;
      if (geom instanceof OlGeomPolygon) {
        tooltipCoord = geom.getInteriorPoint().getCoordinates();
      } else if (geom instanceof OlLineString) {
        tooltipCoord = geom.getLastCoordinate();
      }

     // checking if the tooltip coordinate is in the extent before we render the tooltip and update its position
      const tCoord = transform(tooltipCoord, crs, 'EPSG:4326');
      if (!areCoordinatesWithinExtent(proj, tCoord)) return;

      renderTooltip(feature, tooltipOverlay);
      tooltipOverlay.setPosition(tooltipCoord);
    });
  ```

Reply:
I took a closer look at this and it wasn't as straightforward as I had hoped so I think we are good to stick with your current approach!

Reply:
Sounds good. Once the CI tests all pass I will merge. Thanks.

Reply:
You may want to re-request review from Minnie and get her approval on the PR first